### PR TITLE
Fix: NameError: name 'computer' is not defined

### PR DESCRIPTION
**Pull request by Arrendy**
### Describe the changes you have made:
The `run` method in the `Terminal` class is a generator function that executes some code and yields the results one at a time. In a previous commit, the for loop was removed that was used to iterate over these results, which caused the `run` method to not actually be executed because its results were not being consumed. This commit adds the for loop back to ensure that the `run` method is properly executed.

### Reference any relevant issues (e.g. "Fixes #000"):
https://github.com/KillianLucas/open-interpreter/issues/875

### Pre-Submission Checklist (optional but appreciated):

- [x] I have included relevant documentation updates (stored in /docs)
- [x] I have read `docs/CONTRIBUTING.md`
- [x] I have read `docs/ROADMAP.md`

### OS Tests (optional but appreciated):

- [ ] Tested on Windows
- [x] Tested on MacOS
- [ ] Tested on Linux

### CODE REVIEW

**Comment by Notnaton** [comment]
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "E:\dev\open-interpreter\interpreter\core\core.py", line 46, in start_terminal_interface
    start_terminal_interface(self)
  File "E:\dev\open-interpreter\interpreter\terminal_interface\start_terminal_interface.py", line 543, in start_terminal_interface
    for _ in interpreter.computer.run(
  File "E:\dev\open-interpreter\interpreter\core\computer\terminal\terminal.py", line 40, in run
    for chunk in self._streaming_chat(language, code, stream=True):
                 ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Terminal' object has no attribute '_streaming_chat'

**Comment by Notnaton** [comment]
Added a quick fix, just putting `stream=True`

File: interpreter/terminal_interface/start_terminal_interface.py [hunk 1]
```diff
@@ -541,11 +540,13 @@ def start_terminal_interface(interpreter):
         #         print(chunk.get("content"))
 
         # Give it access to the computer via Python
-        interpreter.computer.run(
-            "python",
-            "import time\nfrom interpreter import interpreter\ncomputer = interpreter.computer",  # We ask it to use time, so
+        for _ in interpreter.computer.run(
```
Comment (KillianLucas) [inline interpreter/terminal_interface/start_terminal_interface.py:543]: This is actually why I added that part that pulls from the stream! It allows you to just run `interpreter.computer.run` without the `for _ in` syntax. I'll merge but will revert this.

File: interpreter/terminal_interface/start_terminal_interface.py [hunk 2]
```diff
@@ -630,7 +631,7 @@ def start_terminal_interface(interpreter):
 
     # Set attributes on interpreter
     for argument_name, argument_value in vars(args).items():
-        if argument_value != None:
+        if argument_value is not None:
```
Comment (KillianLucas) [inline interpreter/terminal_interface/start_terminal_interface.py:634]: ?

**Comment by KillianLucas** [comment]
Thanks @Arrendy! And thanks @Notnaton for working on this as well! This was totally a mistake I made, luckily committed after the last version that went out to `pip`, so just a problem for us working off the repo.

Renaming it to `run` is exactly how it ought to work— if `stream=False`, we just pull from an identical `run` with `stream=True` and return that output. Merging!

File: interpreter/terminal_interface/start_terminal_interface.py (hunk 2 above)
Comment (Arrendy) [reply interpreter/terminal_interface/start_terminal_interface.py:634]: https://peps.python.org/pep-0008/#programming-recommendations

"Comparisons to singletons like None should always be done with is or is not, never the equality operators."

**Pull request merged**